### DISCUSSION TITLE
Что такое или кто такие семь адъютантов разумо-духов?

### DISCUSSION THREAD

**Discussion by tigran123** [tigran123/ud#6] [Идеи и вопросы]
Выбирая термин для описания этих удивительных существ, я решил буквально копировать английскую фразу adjutant mind-spirits, вместо более привычно звучащего варианта <<духов помощников разума>> и сделал это по следующей причине.

Эти существа, с практически автоматической деятельностью которых мы встречаемся при каждом биении сердца и при каждом вдохе и выдохе, не являются духами в смысле возможности личностного контакта с ними со стороны человека. Они вполне соответствуют семи т.н. <<чакрам>> эзотерических учений и проще всего их визуализировать в виде энергетических центров или вихрей, сфокусированных в разных частях тела. На самом деле, их видимые (не обычным зрением, разумеется) проявления в виде <<чакр>> являются лишь проекцией с уровня материального разума на модель нашего тела, создаваемую нашим же разумом, т.е. на ту часть материальной Вселенной, которую мы условно воспринимаем как собственное смертное тело.

Уровень функционирования материального разума, соответствующий каналам адъютантов в точности соответствует уровню восприятия Вселенской космической семьи, как чего-то отдельного от собственного тела и разума. Переход на моронтийный уровень функционирования разума, лежащий _над_ семью адъютантами, даже чисто геометрически (5-е измерение есть собственное время частицы, параметризующее её существование отдельно от других частиц --- <<псевдо-сознание>>, --- т.е. начиная с 5-го измерения Вселенная представляет собой отдельное многообразие для каждой частицы и возможность сосуществования многих частиц означает, приблизительно, изоморфизм всех этих многообразий) подразумевает гармоничное соответствие (но не обязательно отождествление, как и в нашем теле левая и правая рука работают в гармонии друг с другом, но не отождествляются) с другими <<элементами>> космической семьи, достигшими того же уровня восприятия реальности.

На ту же самую нелокальность и нелокализуемость материи (а тем более, разума) указывают соображения из квантовой механики. Например: изменение _квантовых_ траекторий (рассчитанных по квантовой теории движения де Бройля и Бома) частиц, летящих на потенциальный барьер, ещё _до_ достижения барьера, а также смещение интерференционной картины электронных пучков, пролетающих в области нулевого магнитного поля, но отличного от нуля векторного потенциала **А(r)** (эффект Ааронова-Бома).

**Comment by tigran123** [comment]
Я даже не БУКВАЛЬНО копировал английский, а изменил adjutant с прилагательного на существительное, строго говоря. То, что в английском легко передать в виде прилагательного adjutant, в русском требует формы существительного -- иначе получится "вспомогательный".

**Reply by irinachernov** [reply]
Спасибо тебе за это замечательное расширенное объяснение!
